clang/test/Driver/sycl-offload.c

```diff
@@ -598,6 +598,10 @@
 // RUN: %clangxx %s -fsycl -fsycl-explicit-simd -### 2>&1 | FileCheck %s --check-prefix=CHK-FSYCL-ESIMD
 // CHK-FSYCL-ESIMD: llvm-spirv{{.*}}-spirv-allow-unknown-intrinsics
 
+/// Check "-spirv-allow-extra-diexpressions" option is emitted for llvm-spirv tool for debug mode
+// RUN: %clangxx %s -fsycl -g -### 2>&1 | FileCheck %s --check-prefix=CHK-FSYCL-DEBUG
+// CHK-FSYCL-DEBUG: llvm-spirv{{.*}}-spirv-allow-extra-diexpressions
+
```

Comment:
Don't know if this check is useful, given that the option gets enabled for all cases.

Reply:
Right. I have removed this check.